sdk/storage/Azure.Storage.Files.DataLake/samples/Sample02_Auth.cs

```diff
@@ -0,0 +1,198 @@
+
+            // Get a reference to a service Client
+            DataLakeServiceClient service = new DataLakeServiceClient(serviceUri, sharedKeyCredential);
+
+            // Get a reference to a filesystem named "sample-filesystem"
+            DataLakeFileSystemClient filesystem = service.GetFileSystemClient(Randomize("sample-filesystem"));
+            try
+            {
+                // Create a file that can be accessed publicly
+                await filesystem.CreateAsync(PublicAccessType.Container);
+                DataLakeFileClient file = filesystem.GetFileClient(Randomize("sample-file"));
+                await file.CreateAsync();
+
+                // Append data to the file
+                string fileContent = "File Content";
+                await file.AppendAsync(new MemoryStream(Encoding.UTF8.GetBytes(fileContent)), 0);
+                await file.FlushAsync(fileContent.Length);
+
+                // Anonymously access a blob given its URI
+                Uri endpoint = file.Uri;
+                DataLakeFileClient anonymous = new DataLakeFileClient(endpoint);
+
+                // Make a service request to verify we've succesfully authenticated
+                await anonymous.GetPropertiesAsync();
+            }
+            finally
+            {
+                await filesystem.DeleteAsync();
+            }
+        }
+
+        /// <summary>
+        /// Use a shared key to access a Storage Account.
+        ///
+        /// Shared Key authorization relies on your account access keys and
+        /// other parameters to produce an encrypted signature string that is
+        /// passed on the request in the Authorization header.
+        /// </summary>
+        [Test]
+        public async Task SharedKeyAuthAsync()
+        {
+            // Get a Storage account name, shared key, and endpoint Uri.
+            //
+            // You can obtain both from the Azure Portal by clicking Access
+            // Keys under Settings in the Portal Storage account blade.
+            //
+            // You can also get access to your account keys from the Azure CLI
+            // with:
+            //
+            //     az storage account keys list --account-name <account_name> --resource-group <resource_group>
+            //
+            string accountName = StorageAccountName;
+            string accountKey = StorageAccountKey;
+            Uri serviceUri = StorageAccountBlobUri;
+
+            // Make StorageSharedKeyCredential to pass to the serviceClient
+            StorageSharedKeyCredential sharedKeyCredential = new StorageSharedKeyCredential(accountName, accountKey);
+
+            // Get a reference to a service Client
+            DataLakeServiceClient service = new DataLakeServiceClient(serviceUri, sharedKeyCredential);
+
+            // Make a service request to verify we've succesfully authenticated
+            await service.CreateFileSystemAsync("sample-filesystem-sharedkey");
```

Comment:
Is there a `service.GetProperties()` you can call instead of making a call and then cleaning up?

Reply:
There's no service.GetProperties()